api_definition.go

```diff
@@ -205,8 +205,8 @@ func (a *APIDefinitionLoader) MakeSpec(appConfig *apidef.APIDefinition) *APISpec
 		}
 	}
 
-	newAppSpec.RxPaths = make(map[string][]URLSpec)
-	newAppSpec.WhiteListEnabled = make(map[string]bool)
+	newAppSpec.RxPaths = make(map[string][]URLSpec, len(appConfig.VersionData.Versions))
```

Comment:
Yeah. You may call it premature optimization, but it can't hurt. The default value is 0.